src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const items = [...document.querySelectorAll('li')];
+
+items.forEach(item => {
+  const span = document.createElement('span');
+
+  item.prepend(span);
+  span.append(span.nextSibling);
+});
+
+tree.addEventListener('click', e => {
+  const item = e.target.closest('span').nextSibling;
+
+  item.hidden = !item.hidden;
```

Comment:
it would be good to check if `item` isn't `undefined`
```suggestion
  if (item) item.hidden = !item.hidden;
```